sys/net/gnrc/routing/rpl/srh/gnrc_rpl_srh.c

```diff
@@ -41,7 +41,7 @@ int gnrc_rpl_srh_process(ipv6_hdr_t *ipv6, gnrc_rpl_srh_t *rh)
     uint8_t *addr_vec = (uint8_t *) (rh + 1);
     bool found = false;
 
-    DEBUG("RPL SRH: %" PRIu8 " addresses in the routing header\n", n);
+    DEBUG("RPL SRH: %u addresses in the routing header\n", n);
```

Comment:
I'd add a type cast in here, simply because I don't know if the compiler *has to* promote the variable to the word size implicitly. If not, then the upper bytes of the register that stores `n` could contain garbage, that could get printed out.